draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -3584,10 +3584,6 @@ Upgrade: websocket
 <artwork type="example">
   Content-Length: 3495
 </artwork>
-<t>
-   A sender &MUST-NOT; send a Content-Length header field in any message that
-   contains a <x:ref>Transfer-Encoding</x:ref> header field.
-</t>
 <t>
    A user agent &SHOULD; send a Content-Length in a request message when no
    <x:ref>Transfer-Encoding</x:ref> is sent and the request method defines
```

Comment:
The fix for this might be as follows:

>  A user agent `&SHOULD;` send a Content-Length in a request message when the request method defines a meaning for enclosed content.  However, in HTTP/1.1 Content-Length cannot be included when `<x:ref>`Transfer-Encoding`</x:ref>` is also present.